README.md

```diff
@@ -2,8 +2,27 @@
 
 This repository contains the Apache DataFusion blog at https://datafusion.apache.org/blog/
 
+## Testing
+
+There are two ways to preview your blog post before publishing the site. You can
+either locally build and test the site or you can use the auto staging feature
+of the CI system. To locally build the site on your machine, follow the docker
+instructions below.
+
+To use the staging feature of the CI system, push a branch that starts with
+`site/` and create a PR to merge this branch into `main`. When you do so, it
+will trigger a CI process that will build the site and push it to the branch
+`asf-staging`. Once this completes, the ASF infrastructure will auto publish
+this staged branch to https://datafusion.staged.apache.org/
+
+The most recently run staging CI pipeline will be published to this site. If you
```

Comment:
I tested it out and I think this workflow doesn't work for forks -- see review comments

```suggestion
This workflow only works for branches in this repository (not for forks). 
In order to use https://datafusion.staged.apache.org/ you need to be a committer
and create the branch directly in this repository.

The most recently run staging CI pipeline will be published to this site. If you
```